packages/i18n/src/locales/en/options/lunetius.yml

```diff
@@ -13,5 +13,5 @@ length:
     toKnee: On the knee
     toBelowKnee: Below the knee
     toHips: On the hips
-    toUpperLeg: Below the hips
+    toUpperLeg: To the upper leg
```

Comment:
Since `toHips` is `On the hips` and `toKnee` is `On the knee`, perhaps `toUpperLeg` can be `On the thigh`

Reply:
Oh, great idea! Done.